notebooks/13_Theta_phase_and_power.ipynb

```diff
@@ -0,0 +1,1123 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    from spyglass.lfp.v1.analytic_signal import *
</pre><p>I would say it's generally better practice to specify your imports, so it would be great to model that here and list out exactly what you want to import from spyglass.lfp.v1.analytic_signal.</p><p>I think we've also (maybe this is just me) gotten in the habit of not importing standalone tables, but instead import the module <code>import spyglass.lfp.v1 as sg_lfp</code> and then running <code>sg_lfp.LFPBand</code> or <code>sg_lfp.analytic_signal.AnalyticSignalParameters</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/496/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>